README.md

```diff
@@ -37,24 +37,29 @@ To intercept resolvers with mocks execute this app with `GRAPHQL_DEBUG=1` enable
 # API
 
 - `GraphQLComponent(options)` - the component class, which may also be extended. Its options include:
-  - `types` - A string or array of strings representing typeDefs and rootTypes.
-  - `resolvers` - An object containing resolver functions.
-  - `imports` - An optional array of imported components for the schema to be merged with.
-  - `context` - An optional object { namespace, factory } for contributing to context.
-  - `directives` - An optional object containing custom schema directives.
-  - `useMocks` - Enable mocks.
-  - `preserveMockResolvers` - Preserve type resolvers in mock mode.
-  - `mocks` - An optional object containing mock types.
+  - `types` - a string or array of strings representing typeDefs and rootTypes.
+  - `resolvers` - an object containing resolver functions.
+  - `imports` - an optional array of imported components for the schema to be merged with.
+  - `context` - an optional object { namespace, factory } for contributing to context.
+  - `directives` - an optional object containing custom schema directives.
+  - `useMocks` - enable mocks.
+  - `preserveMockResolvers` - preserve type resolvers in mock mode.
+  - `mocks` - an optional object containing mock types.
+  - `dataSource` - a data source instance to make available on `context.dataSources` .
```

Comment:
now that you added support for an array of dataSources:

\`dataSources\` - an array of data source instances to make available on \`context.dataSources\`

same would go for below in terms of the api exposed by a GraphQLComponent instance.